src/views/enterEmail.njk

```diff
@@ -0,0 +1,50 @@
+{% extends 'common/base.njk' %}
+{% set title = 'Enter Email' %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% block backLink %}
+  {{ govukBackLink({ text: "Back", href: "/choose-location" }) }}
+{% endblock %}
+{%block content%}
+
+  <form method="post"action="/validate-enter-email" id="enterEmailForm">
```

Comment:
Minor change, missing some spacing between the `method` and `action` attributes and the nunjucks templating on line 9 and line 50